src/api/flaskr/service/tts/pipeline.py

```
@@ -0,0 +1,411 @@
+
+
+def _concat_mp3_best_effort(parts: Sequence[bytes]) -> bytes:
+    if not parts:
+        return b""
+    if len(parts) == 1:
+        return parts[0]
+
+    if is_audio_processing_available():
+        try:
+            return concat_audio_mp3(list(parts))
+        except Exception as exc:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)